src/PackageArchive/Archive.CiServer.Patch.Compat/ArchiveBaseline.2.1.28.txt

```diff
@@ -0,0 +1,202 @@
+microsoft.visualstudio.web.codegeneration.contracts\2.1.10\.nupkg.metadata
```

Comment:
@dotnet/aspnet-build should these files be in main?

Reply:
No. The entire src/PackageArchive folder should not exist in main